gnd/src/main/java/com/google/android/gnd/model/feature/FeatureMutation.java

```diff
@@ -70,9 +73,10 @@ public static Builder builder() {
   @AutoValue.Builder
   public abstract static class Builder extends Mutation.Builder<Builder> {
 
-    public abstract Builder setNewLocation(Optional<Point> newNewLocation);
+    public abstract Builder setNewLocation(@Nullable Optional<Point> newNewLocation);
```

Comment:
Do we need `@Nullable` if the field is already `Optional`? If it's because you receive errors when the field is not specified before calling `build()`, you can override the `newBuilder()` method to set these two fields to `Optional.empty()` by default. See similar `@AutoValue` classes for an example.

Reply:
removed @Nullable.